src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.test.tsx

```diff
@@ -269,3 +270,151 @@ describe('DataSourceAggregatedView: read active view (displayAllCompatibleDataSo
     }
   );
 });
+
+describe('DataSourceAggregatedView empty state test with local cluster hiding', () => {
+  let component: ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;
+  let client: SavedObjectsClientContract;
+  const { toasts } = notificationServiceMock.createStartContract();
+  const uiSettings = uiSettingsServiceMock.createStartContract();
+  const application = applicationServiceMock.createStartContract();
+  const nextTick = () => new Promise((res) => process.nextTick(res));
+
+  beforeEach(() => {
+    client = {
+      find: jest.fn().mockResolvedValue([]),
```

Comment:
looks like we return empty array no matter what, and are we testing the filter at all?